File: insertion.js
function insertion_sort(vec){
    var j, clave;

    for (var i = 0; i < vec.size(); i++){
        clave = vec[i];
        j = i-1;
        while (j >= 0 && vec[j]>clave){
            vec[j+1] = vec[j];  
            j = j-1;
        }
        vec[j+1] = clave;    
    }
}

File: insertion.py
def InsertionSort(A):
  for i in range(1, len(A)):
    cur_value = A[i]
    j = i - 1
    while j >= 0 and A[j] > cur_value:
      A[j+1] = A[j]
      j -= 1
    A[j+1] = cur_value